inscriptions/ISic003323.xml

```diff
@@ -199,12 +199,12 @@
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#DEE">
                 <ab>
                     <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><persName type="attested"><name type="nomen"><seg part="F"><unclear>i</unclear><unclear>o</unclear></seg></name> <name type="cognomen"><supplied reason="lost">I</supplied><unclear>n</unclear><unclear>n</unclear><unclear>o</unclear><unclear>c</unclear>enti</name></persName>
-                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><gap reason="illegible" quantity="1" unit="character"/>V <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/><gap reason="illegible" quantity="2" unit="character"/><unclear>NIA</unclear><gap reason="illegible" quantity="1" unit="character"/>NIS <g ref="#interpunct">·</g> HV <gap reason="lost" extent="unknown" unit="character"/> <w>obito</w> <g ref="#interpunct">·</g> <w>ex</w>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><orig>v</orig> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/><orig><unclear>t</unclear></orig><orig><unclear>nia</unclear></orig><orig><unclear>nis</unclear></orig> <g ref="#interpunct">·</g> <orig>hu</orig> <gap reason="lost" extent="unknown" unit="character"/> <w>obito</w> <g ref="#interpunct">·</g> <w>ex</w>
```

Comment:
Same as above - you can put these in one `<unclear>` and in one '<orig>' if there are no discernable word divisions.